src/shared_queue.cc

```diff
@@ -49,8 +98,7 @@ uint32_t SharedQueue::registerQueue(std::string_view vm_id, std::string_view que
 
 uint32_t SharedQueue::resolveQueue(std::string_view vm_id, std::string_view queue_name) {
   std::lock_guard<std::mutex> lock(mutex_);
-  auto key = std::make_pair(std::string(vm_id), std::string(queue_name));
-  auto it = queue_tokens_.find(key);
+  auto it = queue_tokens_.find(std::make_pair(std::string(vm_id), std::string(queue_name)));
```

Comment:
This seems like an unnecessary change. Compiler is going to optimize it for you anyway (at least, I hope so), and it makes the code less readable.